src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
```

Comment:
```suggestion
```

Reply:
need remove redundant empty lines in Machine?like 4 line